metric/syncfloat64.go

```diff
@@ -25,6 +25,13 @@ type Float64Counter interface {
 	// Use the WithAttributeSet (or, if performance is not a concern,
 	// the WithAttributes) option to include measurement attributes.
 	Add(ctx context.Context, incr float64, options ...AddOption)
+
+	// IsEnabled returns true if this instrument is enabled.
+	//
+	// This helps users avoid performing computationally expensive operations when
+	// recording measurements. The returned value cannot be cached since it
+	// may change over time.
+	IsEnabled(ctx context.Context, options ...EnabledOption) bool
```

Comment:
The context argument may be removed pending the closing of open-telemetry/opentelemetry-specification/issues/4256

Reply:
From my reading of that spec issue, it looks like the consensus is around not having to explicitly add the context to the spec, but still allow Go to require one.
With that in mind, it looks like the context argument could also be kept?

Reply:
Alright, you definitely have more experience than me reading spec legalese :smile: It was unclear to me based on the comments there, but if it's clear to you I guess I can mark this as solved